stats.mod

```diff
@@ -860,7 +861,8 @@ static double hash (void* vv) {
       } else   {  xx.d=vvo[j][i]; }
       if (xx.i[0]==0) { xx.i[0]=xx.i[1]; xx.i[0]<<=4; } // high order bits may be 0
       if (xx.i[1]==0) { xx.i[1]=xx.i[0]; xx.i[1]<<=4; } // low order bits unlikely 0
-      mcell_ran4_init(&xx.i[1]);
+      /* casts are trying to preserve (probably buggy) C behaviour in C++ */
+      mcell_ran4_init((uint32_t)(uintptr_t)&xx.i[1]);
```

Comment:
Presumably this was not the intended behaviour, but it should at least be the old behaviour...

Reply:
This issue might date back to https://github.com/neuronsimulator/nrn/commit/31f3955f730e1394fed03e640d8288edf088eb72.